extensions/wikia/EditAccount/SpecialEditAccount_body.php

```diff
@@ -227,24 +235,25 @@ public function execute( $par ) {
 
 			// emailStatus is the status of the email in the "Set new email address" field
 			$emailStatus = ( $this->mUser->isEmailConfirmed() ) ? wfMsg('editaccount-status-confirmed') : wfMsg('editaccount-status-unconfirmed') ;
-			$oTmpl->set_Vars( array(
-					'userEmail' => $this->mUser->getEmail(),
-					'userRealName' => $this->mUser->getRealName(),
-					'userId'  => $this->mUser->getID(),
-					'userReg' => date( 'r', strtotime( $this->mUser->getRegistration() ) ),
-					'isUnsub' => $this->mUser->getGlobalPreference('unsubscribed'),
-					'isDisabled' => $this->mUser->getGlobalFlag('disabled'),
-					'isClosureRequested' => $this->isClosureRequested(),
-					'isAdopter' => $this->mUser->getGlobalFlag('AllowAdoption', 1 ),
-					'userStatus' => $userStatus,
-					'emailStatus' => $emailStatus,
-					'changeEmailRequested' => $changeEmailRequested,
-					'mailLogLink' => Linker::linkKnown(
-						SpecialPage::getTitleFor( 'EditAccount', 'log' ),
-						"Mail change log",	// TODO: i18n this
-						array(),			// attribs
-						array('user_id' => $this->mUser->getID())),
-				) );
+			$oTmpl->set_Vars( [
+				'userEmail' => $this->mUser->getEmail(),
+				'userRealName' => $this->mUser->getRealName(),
+				'userId'  => $this->mUser->getID(),
+				'userReg' => date( 'r', strtotime( $this->mUser->getRegistration() ) ),
+				'isUnsub' => $this->mUser->getGlobalPreference('unsubscribed'),
+				'isDisabled' => $this->mUser->getGlobalFlag('disabled'),
+				'isClosureRequested' => $this->isClosureRequested(),
+				'isAdopter' => $this->mUser->getGlobalFlag('AllowAdoption', 1 ),
+				'isFanContributor' => $this->isFanContributor(),
+				'userStatus' => $userStatus,
+				'emailStatus' => $emailStatus,
+				'changeEmailRequested' => $changeEmailRequested,
+				'mailLogLink' => Linker::linkKnown(
+					SpecialPage::getTitleFor( 'EditAccount', 'log' ),
+					"Mail change log",	// TODO: i18n this
+					array(),			// attribs
+					array('user_id' => $this->mUser->getID())),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "user_id" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)